### DISCUSSION TITLE
SMTP notification mail to the wrong address

### DISCUSSION THREAD

**Discussion by Thanatosone** [authelia/authelia#6381] [Troubleshooting] [answered]
### Version

v4.37.5

### Deployment Method

Docker

### Reverse Proxy

NGINX Proxy Manager

### Description

I am really satisfied with the solid funtion to Authelia and really appreciate to the hard work from the author. However reacently I found probably a bug when using notification with SMTP. When it send mail to my Gmail, it always send two mail, one to the address that I assign in the configuration.yml at the same time another mail to "authelia@authelia.com". As a result there's a warning message from Gmail says that it's fail to deliver. It dosn't affect the SMTP funtion since there's still have one correct mail but I am wondering is there anyway to stop it mailing to "authelia@authelia.com"

### Reproduction

[https://i.imgur.com/qNZTl4t.png](url)

  smtp:
    host: smtp.gmail.com
    port: 587
    timeout: 5s
    username: myusername@gmail.com
    password: mypassword
    identifier: localhost
    subject: "[Authelia] {title}"
    startup_check_address: myusername@gmail.com
    disable_require_tls: false
    disable_html_emails: false

### Build Information

```shell
docker run -d --name authelia --restart=always --network=host -v /authelia/config:/config authelia/authelia:latest
```

### Logs (Authelia)

```shell
there's no any error message in log
```

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)

- [X] This is a bug report and not a support request

- [X] I have read the security policy and this bug report is not a security issue or security related issue

- [X] I have either included the complete configuration file or I am sure it's unrelated to the configuration

- [X] I have either included the complete debug / trace logs or the output of the build-info command if the logs are not relevant

- [X] I have provided all of the required information in full with the only alteration being reasonable sanitization in accordance with the [Troubleshooting Sanitization](https://www.authelia.com/r/sanitize) reference guide

- [X] I have checked for related proxy or application logs and included them if available

- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment]
I'm converting this to a discussion, the required information for a bug report was not included and it's also unlikely to be a bug.

**Comment by james-d-elliott** [comment, marked as answer]
We send a single notification email to the users first email address from the authentication backend, either the yaml file (not the config) or the LDAP backed. We do not send any additional emails.

**Reply by Thanatosone** [reply]
Thanks for so quick reply. After read your anser I check the authentication_backend in the yaml file. I think I found the problem that I forgot to modify the email in users_database.yml since I use file as my default authentication. Again thank you so much and sorry to bother you with this kind of obvious mistake.
